src/components/BooksContainer.jsx

```diff
@@ -1,17 +1,73 @@
-// import BookItem from './BookItem';
-import { useSelector } from 'react-redux';
-// import { addBook, removeBook } from './bookSlice';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/bookSlice';
 
 const BooksContainer = () => {
   // useSelector allow us to pick the desired item from a store state value.
   // state.<name of the reducer slice>.<the data you wanna pick>
+
   const books = useSelector((state) => state.books.bookItems);
-  // dispatch allow us to update the state after using reducers and useselectors
-  // const dispatch = useDispatch();
+  //dispatch allow us to update the state after using reducers and useselectors
+
```

Comment:
- [ ] [OPTIONAL] I do feel that your code will be a lot cleaner and impressive if you remove unnecessary comments from the code. Comments should be used as important mark-downs of salient facts about your code capable of guiding you or improving readability and comprehension in the future not just litters of evident facts.